examples/README.md

```diff
@@ -1,10 +0,0 @@
-# Python Examples
-
-These examples are intended to demonstrate how to use Delphyne
-from the python layer. Each script permits the --help argument
-which provides a more detailed explanation of the example's
-functionality and purpose.
-
-The examples only run from within an installed Delphyne
-environment. For more information, read the
-[Delphyne Guide](https://docs.google.com/document/d/1tQ9vDp084pMuHjYmtScLB3F1tdr4iP9w7_OTcoSM1zQ).
```

Comment:
We should not lose track of this spreadsheet. Let's add it somewhere in the readme.

Reply:
Alternatively, it isn't bad to keep this readme here.

Reply:
@stonier do you think we can make the document public with read access only?

Reply:
Solving this comment with a new task in https://github.com/maliput/maliput_documentation/issues/100 per weekly meeting